### ISSUE TITLE
There's a typo in "Predefined Unmarshallers"

### ISSUE THREAD

**Issue by satansk** [akka/akka-http#1422]
In [Predefined Unmarshallers](http://doc.akka.io/docs/akka-http/10.0.10/scala/http/common/unmarshalling.html#predefined-unmarshallers):

```
Akka HTTP already predefines a number of marshallers for the most common types. Specifically these are:
```

Here, the `marshallers` should be `unmarshallers`, and I'll fix it.

**Comment by jrudolph** [comment]
Thanks, good catch. Btw. it's ok to just open a PR to fix typos. ;)

**Comment by satansk** [comment]
@jrudolph thanks and get it 😀

**Comment by jrudolph** [comment]
Fixed by #1423.
